### PR TITLE
Add `mkaczanowski/pastebin` to the pastebin subsection

### PR DESCRIPTION
**Pull request by mkaczanowski**
I propose to add another pastebin service to the list.

I am convinced it's a healthy alternative to what currently exists as explained here: https://github.com/mkaczanowski/pastebin#why

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1179,6 +1179,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [SocksBin](https://github.com/magnumdingusedu/socksbin) - Simple and fast terminal based pastebin, with optional code highlighting. No specific client required, all you need is netcat. [Demo](https://socksbin.magnum.wtf). `GPL-3.0` `Python` 
 - [Stikked](https://github.com/claudehohl/Stikked) - Advanced and beautiful pastebin. ([Demo](https://paste.scratchbook.ch/)) `GPL-3.0` `PHP`
 - [Sup3rS3cretMes5age](https://github.com/algolia/sup3rS3cretMes5age) - Very simple (to deploy and to use) secret message service using Hashicorp Vault as a secrets storage. `MIT` `Go`
+- [pastebin](https://github.com/mkaczanowski/pastebin) - Simple, fast, feature-rich, standalone pastebin service. ([Demo](https://www.youtube.com/watch?v=BG7f61H7C4I), [Source Code](https://github.com/mkaczanowski/pastebin)) `MIT` `Rust`
```
Comment (epsilon-0) [inline README.md:1182]: Don't need the source code in two places.  
You can remove the source code from besides the Demo

Comment (mkaczanowski) [reply README.md]: @epsilon-0 thanks, updated!

**Comment by mkaczanowski** [comment]
@nodiscc done

File: README.md [hunk 2]
```diff
@@ -1179,6 +1179,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [SocksBin](https://github.com/magnumdingusedu/socksbin) - Simple and fast terminal based pastebin, with optional code highlighting. No specific client required, all you need is netcat. [Demo](https://socksbin.magnum.wtf). `GPL-3.0` `Python` 
 - [Stikked](https://github.com/claudehohl/Stikked) - Advanced and beautiful pastebin. ([Demo](https://paste.scratchbook.ch/)) `GPL-3.0` `PHP`
 - [Sup3rS3cretMes5age](https://github.com/algolia/sup3rS3cretMes5age) - Very simple (to deploy and to use) secret message service using Hashicorp Vault as a secrets storage. `MIT` `Go`
+- [pastebin](https://github.com/mkaczanowski/pastebin) - Simple, fast, feature-rich, standalone pastebin service. `MIT` `Rust`
```
Comment (nodiscc) [inline README.md:1182]: Please order the entries alphabetically

Comment (mkaczanowski) [reply README.md]: done

**Comment by mkaczanowski** [comment]
@nodiscc done

**Pull request merged**